libraries/exoplayer/src/main/java/androidx/media3/exoplayer/util/SntpClient.java

```diff
@@ -49,6 +49,9 @@ public final class SntpClient {
   /** The default maximum time, in milliseconds, to wait for the SNTP request to complete. */
   public static final int DEFAULT_TIMEOUT_MS = 1_000;
 
+  /** Max time to allow before re-initializing with NTP server, default is 10 minutes */
+  private static final long MAX_ELAPSED_MS_TILL_UPDATE = 60 * 10 * 1_000L;
```

Comment:
UNTIL ? :)

Reply:
As mentioned in the PR comment. Is this a value that comes from your experience when operating live streams? I'm mostly curious...